docs/smartcontracts.md

```diff
@@ -24,30 +24,17 @@ In this guide, we use a straightforward "counter" smart contract to illustrate h
 
 The counter contract has two entry points named `increment` and `decrement.` Taquito uses these entrypoints to generate corresponding javascript methods available to the developer.
 
-The counter contracts storage is a simple integer that gets increased or decreased based on the calls to the entrypoints.
+The counter contract's storage is a simple integer that gets increased or decreased based on the calls to the entrypoints.
 
-### Counter Contract in CameLIGO
+### Counter Contract in JSLIGO
 
-```ocaml
-type storage = int
+```js
+type storage = int;
 
-(* variant defining pseudo multi-entrypoint actions *)
-
-type action =
-| Increment of int
-| Decrement of int
-
-let add (a,b: int * int) : int = a + b
-let sub (a,b: int * int) : int = a - b
-
-(* real entrypoint that re-routes the flow based on the action provided *)
-
-let main (p,s: action * storage) =
- let storage =
-   match p with
-   | Increment n -> add (s, n)
-   | Decrement n -> sub (s, n)
- in ([] : operation list), storage
+// @entry
+const increment = (p : int, s : storage) : [list<operation>, storage] => [list([]), s + p];
+// @entry
+const decrement = (p : int, s : storage) : [list<operation>, storage] => [list([]), s - p];
```

Comment:
is it just github? or is `main` and the `action` type gone? 
all i see is the storage type and the entrypoint functions

Reply:
yeah, you are right.  he did not include that in the request, I guess he assumed I would add it.